lib/colorado_lottery.rb

```diff
@@ -0,0 +1,21 @@
+class ColoradoLottery
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = ({})
```

Comment:
No need for the outer parens `()` on these hashes. Only need those if we are passing a hash to a minitest assertion method.